cric-exchange-app-server/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import {createConnection} from "typeorm";
```

Comment:
Can we rename this file to User.ts?

Reply:
There is already a file name User.ts . Do you mean rename the import?

Reply:
Isn't this index.ts?
but nvm i see that this a temp thing